plugins/woocommerce-beta-tester/src/live-branches/components/BranchList.tsx

```diff
@@ -188,7 +190,7 @@ export const BranchList = ( { branches }: { branches: Branch[] } ) => {
 				</CardBody>
 				<CardFooter></CardFooter>
 			</Card>
-			{ installedBranches.length && (
+			{ installedBranchesExist && (
```

Comment:
This fixes a bug where the character `0` was displayed when no installed branches existed.

Reply:
In this case, does it make sense to coerce to a number?

Reply:
@psealock Not sure what you mean. array.length is a number already. it should be coerced from number to boolean because thats why a 0 was printed. since length is `0`

You could also do this code inline, but I thought maybe storing it in a variable was clearer.

Here's a code example of the behaviour of `number` and `boolean` in React: https://codesandbox.io/s/happy-wind-lbuo8v?file=/src/App.tsx

Reply:
Ok, I read "character" to mean string. All good